### PR TITLE
Support Escape DomKey

### PR DESCRIPTION
**Pull request by mgiuffrida**
Addresses issue #38.

### CODE REVIEW

**Comment by valdrinkoshi** [comment]
Hi @mgiuffrida, thanks for the contribution! Please add a unit test for this and is good to merge :)

File: iron-a11y-keys-behavior.html [hunk 1]
```diff
@@ -102,6 +109,8 @@
         var lKey = key.toLowerCase();
         if (lKey === ' ' || SPACE_KEY.test(lKey)) {
           validKey = 'space';
+        } else if (ESC_KEY.test(lKey)) {
```
Comment (danbeam) [inline iron-a11y-keys-behavior.html]: this code is basically equivalent to `lKey === 'escape'`, right?

did you mean to add an `/i` after the regex, as it seems like the linked URL actually says `Escape` (uppercase E)?

File: iron-a11y-keys-behavior.html [hunk 2]
```diff
@@ -102,6 +109,8 @@
         var lKey = key.toLowerCase();
```
Comment (danbeam) [inline iron-a11y-keys-behavior.html]: ah, missed this `toLowerCase()`

**Comment by mgiuffrida** [comment]
The PR has been updated to include the [variations on KeyboardEvent.key for Escape](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/key#Key_values), so that PR has to land first.

Does there need to also be a version update to do this right?

**Comment by valdrinkoshi** [comment]
Hi @mgiuffrida, we'll handle the version update after the PR gets merged.
LGTM

**Comment by mgiuffrida** [comment]
These tests will fail if PolymerElements/iron-test-helpers/pull/52 isn't landed first...

**Comment by valdrinkoshi** [comment]
@mgiuffrida this got already merged and published. It's my fault, didn't notice your tests are passing also on master :x

Could you open a new PR & update the tests like this? https://github.com/PolymerElements/iron-a11y-keys-behavior/blob/master/test/basic-test.html#L193
```javascript
    test('handles Escape', function() {
      var event = new CustomEvent('keydown');
      event.key = 'Escape';
      keys.dispatchEvent(event);
      expect(keys.keyCount).to.be.equal(1);
    });

**Comment by valdrinkoshi** [comment]
@mgiuffrida update: the tests pass because you also pass the `keyCode` :stuck_out_tongue: 
The fix should be to pass `null` as `keyCode`, that should do!

**Comment by mgiuffrida** [comment]
My bad, I thought I confirmed the test failed but must've gotten confused in the repo shuffling. PTAL at PR #40.

**Comment by valdrinkoshi** [comment]
Awesome, thanks again :)

**Pull request merged**